src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -15,14 +15,17 @@
 
 error_reporting(-1);
 
-if (PHP_VERSION_ID >= 70200) {
-    // PHPUnit 6 is required for PHP 7.2+
+if (PHP_VERSION_ID >= 70100) {
+    // PHPUnit 7 is required for PHP 7.1+
```

Comment:
The comment is not accurate: PHPUnit 7 is required for PHP 7.3+. On php 7.1, we could very well use PHPUnit 5 or 6, so PHPUnit 7 is not a requirement here.

That being said, I agree that we should default to PHPUnit 7 on PHP 7.1 already.

Reply:
Changed